src/Nethermind/Nethermind.Runner/Properties/launchSettings.json

```diff
@@ -112,6 +112,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
+    "Mekong": {
```

Comment:
Please move this below Mainnet so items are alphabetically sorted.

Reply:
fixed